apps/dc/dc-purview-shir/dc-purview-shir.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: dc-purview-shir-deployment
+  namespace: dc
+spec:
+  interval: 5m
+  releaseName: dc-purview-shir
+  values:
+    labels:
+      app.kubernetes.io/instance: dc-purview-shir-deployment
+      app.kubernetes.io/name: dc-purview-shir-deployment
```

Comment:
I don't think this is supported but should just be added automatically so remove
```suggestion
```

Reply:
Changes made